nuxeo-runtime/nuxeo-stream/src/test/java/org/nuxeo/lib/stream/tests/tools/TestToolsChronicle.java

```diff
@@ -64,6 +69,17 @@ public void createContent() throws Exception {
 
     }
 
+    @Override
+    public void testSeek() {
+        super.testSeek();
+        try {
+            run(String.format("seek %s --log-name %s --group anotherGroup --timestamp %s", getManagerOptions(), LOG_NAME, Instant.now().minus(1, ChronoUnit.HOURS)));
+            fail();
+        } catch (UnsupportedOperationException uoe) {
+            assertNotNull(uoe);
```

Comment:
`uoe` is never going to be `null` here. Maybe assert on `uoe.getMessage()`?

Reply:
Yes, I just wanted to assert something to show that's the correct path.

Reply:
I sometimes just use `// ok` for this. But asserting on the exception message is better (unless it changes a lot or is really not relevant)